experimental/packages/opentelemetry-sdk-metrics-base/src/export/PeriodicExportingMetricReader.ts

```diff
@@ -62,7 +63,16 @@ export class PeriodicExportingMetricReader extends MetricReader {
 
   private async _runOnce(): Promise<void> {
     const metrics = await this.collect({});
-    await this._exporter.export(metrics);
+    await this._exporter.export(metrics, result => {
```

Comment:
We should promisify the export callback to get the `await` work correctly, right?

Reply:
This was appropriate earlier but it's unnecessary now. Refactored this a little, please take another look.